sources/packages/api/src/services/application/application.models.ts

```diff
@@ -1,6 +1,12 @@
-import { Application } from "../../database/entities";
+import { Application, StudentAssessment } from "../../database/entities";
 
 export interface ApplicationOverriddenResult {
   overriddenApplication: Application;
   createdApplication: Application;
+  createdAssessment: StudentAssessment;
+}
+
+export interface ApplicationSubmissionResult {
+  application: Application;
+  assessment: StudentAssessment;
 }
```

Comment:
Are we also creating database entities as interfaces ?

Reply:
Sorry, I am not following. This is a service model used to return a result from a service method. Can we please elaborate more?

Reply:
Oh ! never mind, I thought it was in database entities. Resolving comment.